content/en/real_user_monitoring/dashboards/_index.md

```diff
@@ -10,22 +10,19 @@ further_reading:
 
 ## Overview
 
-When you create a RUM application, Datadog [collects data][1] and generates dashboards about your application's performance, errors, resources, user sessions, and mobile performance. 
+When you create a RUM application, Datadog [collects data][1] and generates dashboards about your application's performance, errors, resources, and user sessions. 
 
-{{< img src="real_user_monitoring/dashboards/rum_application_overview_dashboard.png" alt="RUM Application Overview page" style="width:90%;" >}}
+{{< img src="real_user_monitoring/dashboards/rum_application_overview_dashboard-3.png" alt="RUM Application Overview page" style="width:90%;" >}}
 
 Access your RUM dashboards by filtering for `RUM` in the search query of the [**Dashboard List**][2] or by clicking on a dashboard's **Application Overview** in the [**RUM Applications** page][3].
```

Comment:
can we rephrase the second part of these instructions? something like:
... or from your application summary pages (**Real User Monitoring > Performance Summary** and **Real User Monitoring > Analytics Summary**)

not sure what our standard link syntax is, but that's the gist